package.json

```diff
@@ -42,6 +42,7 @@
     "url": "https://github.com/netlify/cli/issues"
   },
   "scripts": {
+    "noop": "",
```

Comment:
This is used for `netlify-lambda` tests, which need to execute an actual npm script. We run this script which is basically a no-op.

Reply:
Removed this for now. We're currently only checking whether `netlify-lambda` is detected, not whether the corresponding npm command runs. To do that, we'll need to either mock `execa` or find a creative solution for making the command available in the test fixture.